chain/chain/src/chain.rs

```diff
@@ -367,7 +371,22 @@ impl Chain {
         last_known_hash: &CryptoHash,
     ) -> Result<CryptoHash, Error> {
         let bps = runtime_adapter.get_epoch_block_producers_ordered(&epoch_id, last_known_hash)?;
-        Chain::compute_bp_hash_inner(bps.iter().map(|(bp, _)| bp).cloned().collect::<Vec<_>>())
+        #[cfg(not(feature = "protocol_feature_block_header_v3"))]
```

Comment:
there is a macro `checked_feature` that you can use

Reply:
I don't think I should use the macro in this case because the macro requires being passed the protocol version. The protocol version is not already present in the scope of this function, but it can be looked up because we have the epoch information present. However, in the case `protocol_feature_block_header_v3` is not enabled the protocol version does not need to be looked up, so it would be doing extra work for no reason.